src/store/getters/state-getters.ts

```diff
@@ -692,4 +717,27 @@ export const getFilingName = (state: StateIF): FilingNames => {
   if (isCorrectionFiling(state)) return FilingNames.CORRECTION
   if (isAlterationFiling(state)) return FilingNames.ALTERATION
   if (isChangeFiling(state)) return FilingNames.CHANGE_OF_REGISTRATION
+  return null
+}
+
+/** The completing party data. */
+export const getCompletingParty = (state: StateIF): CompletingPartyIF => {
+  const mailingAddress = getOrgInfo(state)?.mailingAddress
+
+  return {
+    firstName: getUserFirstName(state) || '',
+    lastName: getUserLastName(state) || '',
+    mailingAddress: {
+      addressCity: mailingAddress?.city || '',
+      addressCountry: mailingAddress?.country || '',
+      addressRegion: mailingAddress?.region || '',
+      addressType: 'mailing',
+      deliveryInstructions: '',
+      postalCode: mailingAddress?.postalCode || '',
+      streetAddress: mailingAddress?.street || '',
+      streetAddressAdditional: mailingAddress?.streetAdditional || ''
+    },
+    email: getUserEmail(state) || '',
+    phone: getUserPhone(state) || ''
```

Comment:
We might not use email or phone atm but we have the data so I thought I'd include it here. (It was hard to untangle how other UIs fetched this data so I made it clear here 😉 )